synapse/lib/layer.py

```diff
@@ -1188,6 +1188,9 @@ async def __anit__(self, core, layrinfo):
 
         self.fresh = not os.path.exists(path)
 
+        if self.fresh and self.readonly:
```

Comment:
Maybe it's ok to create a new readonly layer but we should create the LMDB file anyway

Reply:
( this was originally from the remote layer capability )